src/Dapr.Client/UnsubscribeConfigurationResponse.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+namespace Dapr.Client
+{
+    /// <summary>
+    /// Response from an Unsubscribe Configuration call.
+    /// </summary>
+    public class UnsubscribeConfigurationResponse
+    {
+        /// <summary>
+        /// Boolean representing if the request was successful or not.
+        /// </summary>
+        public bool Ok { get; }
+
+        /// <summary>
+        /// The message from the Configuration API.
+        /// </summary>
+        public string Message { get; }
```

Comment:
This seems confusing with the `Ok` and `Message` fields. What would happen if we threw an exception on failure? That might be more idiomatic.

Reply:
I can add an exception for it, but this is also what the wire returns. 

Thoughts on adding an exception if we get Ok == false and instead throwing a `DaprException(Message)` @rynowak?